### PR TITLE
node/gov: Adjust chain parameters

### PR DESCRIPTION
**Pull request by tbjump**
This is an implementation of https://github.com/wormhole-foundation/wormhole/discussions/3259

How to review: Check that this PR matches the consensus in the discussion link.

### CODE REVIEW

**Comment by tbjump** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3260** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3260" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3260?utm_source=stack-comment).

File: node/pkg/governor/mainnet_chains.go [hunk 1]
```diff
@@ -10,30 +10,30 @@ import (
...
-		chainConfigEntry{emitterChainID: vaa.ChainIDTerra2, dailyLimit: 500_000, bigTransactionSize: 50_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDMoonbeam, dailyLimit: 5_000_000, bigTransactionSize: 500_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDArbitrum, dailyLimit: 200_000, bigTransactionSize: 20_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDOptimism, dailyLimit: 200_000, bigTransactionSize: 20_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDBase, dailyLimit: 200_000, bigTransactionSize: 20_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDAptos, dailyLimit: 5_000_000, bigTransactionSize: 500_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDXpla, dailyLimit: 200_000, bigTransactionSize: 20_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDSui, dailyLimit: 5_000_000, bigTransactionSize: 500_000},
-		chainConfigEntry{emitterChainID: vaa.ChainIDSei, dailyLimit: 200_000, bigTransactionSize: 20_000},
+		{emitterChainID: vaa.ChainIDSolana, dailyLimit: 25_000_000, bigTransactionSize: 2_500_000},
```
Comment (claudijd) [inline node/pkg/governor/mainnet_chains.go:13]: Slight nit that their is two change sets happening here, which clouds the actual parameter changes.

Could we move the chainConfigEntry implicits a separate commit to help reviewers see more clearly what values are changing?

Comment (tbjump) [reply node/pkg/governor/mainnet_chains.go:13]: done

**Review by claudijd — APPROVED** [review]

**Review by bruce-riley — APPROVED** [review]
